.gitlab/kitchen_deploy.yml

```diff
@@ -58,18 +62,36 @@ deploy_deb_testing-a6:
     - echo "$APT_SIGNING_KEY_PASSPHRASE" | deb-s3 upload -c "pipeline-$DD_PIPELINE_ID" -m 6 -b $DEB_TESTING_S3_BUCKET -a amd64 --sign=$DEB_GPG_KEY_ID --gpg_options="--passphrase-fd 0 --batch --digest-algo SHA512" --preserve_versions --visibility public $OMNIBUS_PACKAGE_DIR/datadog-signing-keys_${DD_PIPELINE_ID}.deb
     - echo "$APT_SIGNING_KEY_PASSPHRASE" | deb-s3 upload -c "pipeline-$DD_PIPELINE_ID" -m 6 -b $DEB_TESTING_S3_BUCKET -a x86_64 --sign=$DEB_GPG_KEY_ID --gpg_options="--passphrase-fd 0 --batch --digest-algo SHA512" --preserve_versions --visibility public $OMNIBUS_PACKAGE_DIR/datadog-signing-keys_${DD_PIPELINE_ID}.deb
 
-
-deploy_deb_testing-a7:
+deploy_deb_testing-a6_arm64:
   rules:
-    !reference [.on_default_kitchen_tests_a7]
+    !reference [.on_all_builds_a6]
```

Comment:
all builds =/=> kitchen tests run (eg. if you run `inv pipeline.run --all-builds --no-kitchen-tests` on a branch). I think you'll need a new rule that checks that both the all builds and the kitchen tests options are selected.